guides/common/assembly_preparing-environment-for-installation.adoc

```diff
@@ -17,7 +17,7 @@ include::modules/ref_supported-operating-systems.adoc[leveloffset=+1]
 
 include::modules/ref_supported-browsers.adoc[leveloffset=+1]
 
-include::modules/ref_satellite-ports-and-firewalls-requirements.adoc[leveloffset=+1]
+include::modules/ref_ports-and-firewall-requirements.adoc[leveloffset=+1]
```

Comment:
These are fine, just keep in mind that changing reference "anchors" will ultimately break links, so we need to to this via two stages: https://github.com/theforeman/foreman-documentation/pull/416